torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1003,6 +1003,13 @@ def _generate_reduction_kwargs(ndim, supports_multiple_dims=True):
     # Test reducing inner and outer most dimensions
     yield {'dim': 0, 'keepdim': True}
     yield {'dim': -1, 'keepdim': False}
+    # Reductions of CSR tensors use different implementations for
+    # inner and/or outer dimensions. So, as a minimum of testing CSR
+    # implementations the following kwargs must be generated:
+    #   dict(dim=0, keepdim=True)
+    #   dict(dim=1, keepdim=True)
+    #   dict(dim=(0, 1), keepdim=True)
+    yield {'dim': -1, 'keepdim': True}
```

Comment:
This make sense, but it's annoying that *all* reduction tests will be updated. Make we can constrain this to `sample_inputs_masked_reduction` only for now?

Reply:
The constraint is now implemented within `sample_inputs_sparse_csr_masked_reduction`.